scripts/data_files/config-options-current.txt

```diff
@@ -0,0 +1,219 @@
+MBEDTLS_HAVE_TIME
+MBEDTLS_HAVE_TIME_DATE
+MBEDTLS_HMAC_DRBG_C
+MBEDTLS_IGNORE_RETURN
+MBEDTLS_LMS_C
+MBEDTLS_LMS_PRIVATE
+MBEDTLS_MD_C
+MBEDTLS_MEMORY_ALIGN_MULTIPLE
+MBEDTLS_MEMORY_BACKTRACE
+MBEDTLS_MEMORY_BUFFER_ALLOC_C
+MBEDTLS_MEMORY_DEBUG
+MBEDTLS_MPI_MAX_SIZE
+MBEDTLS_MPI_WINDOW_SIZE
+MBEDTLS_NIST_KW_C
+MBEDTLS_NO_64BIT_MULTIPLICATION
+MBEDTLS_NO_UDBL_DIVISION
+MBEDTLS_PEM_PARSE_C
+MBEDTLS_PEM_WRITE_C
+MBEDTLS_PKCS5_C
+MBEDTLS_PK_C
+MBEDTLS_PK_PARSE_C
+MBEDTLS_PK_PARSE_EC_COMPRESSED
+MBEDTLS_PK_PARSE_EC_EXTENDED
+MBEDTLS_PK_WRITE_C
+MBEDTLS_PLATFORM_C
+MBEDTLS_PLATFORM_CALLOC_MACRO
+MBEDTLS_PLATFORM_EXIT_ALT
+MBEDTLS_PLATFORM_EXIT_MACRO
+MBEDTLS_PLATFORM_FPRINTF_ALT
+MBEDTLS_PLATFORM_FPRINTF_MACRO
+MBEDTLS_PLATFORM_FREE_MACRO
+MBEDTLS_PLATFORM_GMTIME_R_ALT
+MBEDTLS_PLATFORM_MEMORY
+MBEDTLS_PLATFORM_MS_TIME_ALT
+MBEDTLS_PLATFORM_MS_TIME_TYPE_MACRO
+MBEDTLS_PLATFORM_NO_STD_FUNCTIONS
+MBEDTLS_PLATFORM_NV_SEED_ALT
+MBEDTLS_PLATFORM_NV_SEED_READ_MACRO
+MBEDTLS_PLATFORM_NV_SEED_WRITE_MACRO
+MBEDTLS_PLATFORM_PRINTF_ALT
+MBEDTLS_PLATFORM_PRINTF_MACRO
+MBEDTLS_PLATFORM_SETBUF_ALT
+MBEDTLS_PLATFORM_SETBUF_MACRO
+MBEDTLS_PLATFORM_SETUP_TEARDOWN_ALT
+MBEDTLS_PLATFORM_SNPRINTF_ALT
+MBEDTLS_PLATFORM_SNPRINTF_MACRO
+MBEDTLS_PLATFORM_STD_CALLOC
+MBEDTLS_PLATFORM_STD_EXIT
+MBEDTLS_PLATFORM_STD_EXIT_FAILURE
+MBEDTLS_PLATFORM_STD_EXIT_SUCCESS
+MBEDTLS_PLATFORM_STD_FPRINTF
+MBEDTLS_PLATFORM_STD_FREE
+MBEDTLS_PLATFORM_STD_MEM_HDR
+MBEDTLS_PLATFORM_STD_NV_SEED_FILE
+MBEDTLS_PLATFORM_STD_NV_SEED_READ
+MBEDTLS_PLATFORM_STD_NV_SEED_WRITE
+MBEDTLS_PLATFORM_STD_PRINTF
+MBEDTLS_PLATFORM_STD_SETBUF
+MBEDTLS_PLATFORM_STD_SNPRINTF
+MBEDTLS_PLATFORM_STD_TIME
+MBEDTLS_PLATFORM_TIME_ALT
+MBEDTLS_PLATFORM_TIME_MACRO
+MBEDTLS_PLATFORM_TIME_TYPE_MACRO
+MBEDTLS_PLATFORM_VSNPRINTF_ALT
+MBEDTLS_PLATFORM_VSNPRINTF_MACRO
+MBEDTLS_PLATFORM_ZEROIZE_ALT
+MBEDTLS_PRINTF_MS_TIME
+MBEDTLS_PSA_ASSUME_EXCLUSIVE_BUFFERS
+MBEDTLS_PSA_BUILTIN_GET_ENTROPY
+MBEDTLS_PSA_CRYPTO_BUILTIN_KEYS
+MBEDTLS_PSA_CRYPTO_C
+MBEDTLS_PSA_CRYPTO_CLIENT
+MBEDTLS_PSA_CRYPTO_EXTERNAL_RNG
+MBEDTLS_PSA_CRYPTO_KEY_ID_ENCODES_OWNER
+MBEDTLS_PSA_CRYPTO_PLATFORM_FILE
+MBEDTLS_PSA_CRYPTO_RNG_HASH
+MBEDTLS_PSA_CRYPTO_RNG_STRENGTH
+MBEDTLS_PSA_CRYPTO_SPM
+MBEDTLS_PSA_CRYPTO_STORAGE_C
+MBEDTLS_PSA_CRYPTO_STRUCT_FILE
+MBEDTLS_PSA_DRIVER_GET_ENTROPY
+MBEDTLS_PSA_ITS_FILE_C
+MBEDTLS_PSA_KEY_SLOT_COUNT
+MBEDTLS_PSA_KEY_STORE_DYNAMIC
+MBEDTLS_PSA_P256M_DRIVER_ENABLED
+MBEDTLS_PSA_RNG_RESEED_INTERVAL
+MBEDTLS_PSA_STATIC_KEY_SLOTS
+MBEDTLS_PSA_STATIC_KEY_SLOT_BUFFER_SIZE
+MBEDTLS_RSA_GEN_KEY_MIN_BITS
+MBEDTLS_RSA_NO_CRT
+MBEDTLS_SELF_TEST
+MBEDTLS_SHA256_SMALLER
+MBEDTLS_SHA256_USE_ARMV8_A_CRYPTO_IF_PRESENT
+MBEDTLS_SHA256_USE_ARMV8_A_CRYPTO_ONLY
+MBEDTLS_SHA512_SMALLER
+MBEDTLS_SHA512_USE_A64_CRYPTO_IF_PRESENT
+MBEDTLS_SHA512_USE_A64_CRYPTO_ONLY
+MBEDTLS_TEST_CONSTANT_FLOW_MEMSAN
+MBEDTLS_TEST_CONSTANT_FLOW_VALGRIND
+MBEDTLS_TEST_HOOKS
+MBEDTLS_THREADING_ALT
+MBEDTLS_THREADING_C
+MBEDTLS_THREADING_PTHREAD
+PSA_CRYPTO_CONFIG_H
```

Comment:
The presence of the header guard is preexisting. I didn't consider it a problem in https://github.com/Mbed-TLS/mbedtls-framework/pull/196 and I still don't. It has very little practical consequence: users will be allowed to define `PSA_CRYPTO_CONFIG_H` in their crypto config file, which is exactly what would happen if the macro was omitted when constructing the list of options. In Mbed TLS, users will not be allowed to define `PSA_CRYPTO_CONFIG_H` if the crypto config file doesn't define it, which is actually kind of desirable even though it doesn't really matter.